lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -354,7 +376,7 @@ def present_strategy(compute_api, wished_server):
         if compute_api.module.check_mode:
             return changed, {"status": "Server %s attributes would be changed." % target_server["id"]}
 
-        server_change_attributes(compute_api=compute_api, target_server=target_server, wished_server=wished_server)
+        target_server = server_change_attributes(compute_api=compute_api, target_server=target_server, wished_server=wished_server)
```

Comment:
I would rather use another variable name, maybe `created_server` ? `server_change_attributes` method should also use another variable name for the returned object.

Reply:
Actually, I can't modify only this variable as it is returned by the function. 
The function will return `target_server` for existing server or updated one. Changing only this variable will cause an undefined variable error on return.